Adding date by default.

jwbos opened the pull request.

kmccurley reviewed (approved):
I'm skeptical that we need another package dependency for this. We could just put \today into the footer and let the author use a package to modify how that is formatted if they want to.

jwbos commented:
> I'm skeptical that we need another package dependency for this. We could just put \today into the footer and let the author use a package to modify how that is formatted if they want to.

Good point, I fixed this and added the option to omit displaying the date when an empty string is passed.
